QuizAppNew/View/HomeView.swift

```diff
@@ -68,4 +69,24 @@ struct HomeView: View {
             })
         }
     }
+    
+    func nazmul() {
+    
```

Comment:
why did you set function name   **nazmul**?

Reply:
I like to use Nazmul  ....